projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.ts

```diff
@@ -165,6 +169,16 @@ export class RasterSymbologyEditorComponent implements OnChanges, OnDestroy, Aft
         return this.defaultColor;
     }
 
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/221540142-31543215-b847-4e6c-9357-e74ad0a89e5b.png)

When modifying the color table of the linear gradient, a new one is calculated instead.
But the table shows my modification.

Reply:
I'm not sure about this one, can you please check if the problem still exists now? I think it should be fixed along with other issues in 142e1baeb77c0acfdc384be8ac8a2c5518d8f462

Reply:
When a user selects linear gradient, it is unclear what happens if you click apply.

Currently, the color table is ignored and a new liner gradient from the "Update color map" section is shown.

But the expectation could also be that the color table is used for populating the gradient.

One option would be to add a button to "Update color map" that creates a new color table.
But when you click apply, the color table is used.